charts/substrate-faucet/templates/configmap.yaml

```
@@ -7,9 +7,8 @@ data:
   {{- range $key, $val := .Values.faucet.config }}
   {{ $key }}: {{ $val | quote }}
   {{- end }}
-  POSTGRESQL_HOST: "{{ .Release.Name }}-postgresql"
-  POSTGRESQL_PORT: "5432"
-  POSTGRESQL_USERNAME: postgres
-  POSTGRESQL_DATABASE: postgres
-  POSTGRESQL_SSLMODE: disable
+  SMF_CONFIG_DB_HOST: "{{ .Release.Name }}-postgresql"
+  SMF_CONFIG_DB_PORT: "5432"
+  SMF_CONFIG_DB_USERNAME: postgres
+  SMF_CONFIG_DB_DATABASE_NAME: faucet
```

```suggestion
  SMF_CONFIG_DB_DATABASE_NAME: {{ .Values.postgresql.global.postgresql.auth.database }} 
```

the default database created by bitnami chart is `postgres`, to override it you need to add
 `postgresql.global.postgresql.auth.database: "faucet"` in `values.yaml`